templates/search/blocks/featured-search__results-block.html

```diff
@@ -7,10 +7,12 @@ <h2 class="featured-search__heading">{{ bucket.label }}</h2>
             <li class="featured-search__results-list-item">
                 <a href="{% record_url record %}" class="featured-search__results-link">
                     <h3 class="featured-search__results-heading">
-                        {{ record|record_detail:"summaryTitle"|striptags }}
+                        {{ record.summary_title }}
                     </h3>
                 </a>
-                <p>Catalogue reference: {{ record|record_detail:"referenceNumber"}}</p>
+                {% if record.reference_number %}
```

Comment:
New `if` condition is added.  Shouldn't `Catalogue reference` show irrespective of the availability of `reference_number`?

Reply:
Resolved in 9672286